Source/Scene/LabelCollection.js

```diff
@@ -908,13 +909,21 @@ LabelCollection.prototype.update = function (frameState) {
     billboardCollection.textureAtlas = this._textureAtlas;
   }
 
+  const that = this;
   if (!defined(this._backgroundTextureAtlas)) {
     this._backgroundTextureAtlas = new TextureAtlas({
       context: context,
       initialSize: whitePixelSize,
     });
     backgroundBillboardCollection.textureAtlas = this._backgroundTextureAtlas;
-    addWhitePixelCanvas(this._backgroundTextureAtlas, this);
```

Comment:
This looks like a bugfix in addition to Promise update? Is that true? If so, what bug does it fix and should we include that in CHANGES? (even if we have no issue number for it).

I noticed a few places where there are little changes like this that appear to be bug fixes of some kind. If we just figured out the code is wrong but don't know how/if a bug manifested from it, that's fine too, nothing to report. But I just wanted to point it out in case.

Reply:
These were to address cases where the code assumed a promise was executing synchronously. When transitioning to native promises, even in cases where the promise immediately resolved, it wouldn't do so until the end of the frame. So the timing of the rest of the function is thrown off. I don't think these fix any existing bugs, just keep behavior working as before.